src/plugins/cams/datasource.py

```
@@ -0,0 +1,509 @@
+        end_date: Optional[datetime] = None,
+        limit: Optional[int] = None,
+        **filters: Any
+    ) -> AsyncIterator[List[Measurement]]:
+        """Get measurements for a sensor (location + parameter).
+        
+        Yields batches of measurements from CAMS gridded data.
+        """
+        try:
+            # Default to last 7 days if no dates specified
+            if start_date is None:
+                start_date = datetime.now() - timedelta(days=7)
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)